dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/export/relationship/RelationshipOperationParamsMapperTest.java

```diff
@@ -112,8 +129,19 @@ void shouldThrowNotFoundExceptionWhenTrackedEntityIsNotFound() {
   }
 
   @Test
-  void shouldMapEnrollmentWhenAEnrollmentIsPassed() throws NotFoundException {
+  void shouldThrowForbiddenExceptionWhenUserHasNoAccessToTrackedEntity() {
```

Comment:
nit: and also up to you, I don't spell out the type of exception as that is apparent from the assertion. So I would name it `shouldThrow...` or `shouldFailToMap...`

Reply:
Done